app/javascript/packs/MainApp/components/Referees.jsx

```diff
@@ -4,6 +4,22 @@ import axios from 'axios'
 import { Header } from 'semantic-ui-react'
 import RefereeTable from './RefereeTable'
 
+const getRefereeName = (firstName, lastName) => {
```

Comment:
Yea let's create a function that can be imported to separate classes, preferably a top level utilities folder.